packages/modules/Extension-Ads/src/contentScript/rate-notification.js

```diff
@@ -0,0 +1,171 @@
+export class RateNotification {
+  constructor() {
+    chrome.runtime.onMessage.addListener(this.handleMessage.bind(this));
+  }
+
+  handleMessage(message, sender, sendResponse) {
+    if (message.action === 'SHOW_NOTIFICATION') {
```

Comment:
We need to establish an internal action naming convention to easily identify by event name the sender/receiver service and whether it is a request or a response.